metagraph/src/graph/alignment/aligner_cigar.hpp

```diff
@@ -66,6 +67,9 @@ class Cigar {
         cigar_.insert(it, std::forward<Args>(args)...);
     }
 
+    template <typename... Args>
+    void erase(Args&&... args) { cigar_.erase(std::forward<Args>(args)...); }
+
     value_type& front() { return cigar_.front(); }
     value_type& back() { return cigar_.back(); }
     const value_type& front() const { return cigar_.front(); }
```

Comment:
so you allow doing anything with `cigar_`. Just add a getter, and then you don't need to write all these begin/end/insert/erase/...

Reply:
I cleaned this up a bit, but I still kept the `begin() const` and `end() const` to make it easier to iterate through a `Cigar` object